src/main/java/org/octri/clinchit/domain/ClinicianReview.java

```diff
@@ -22,11 +22,19 @@
 	@ManyToOne
 	private SdhDomain domain;
 	
+	@NotNull
 	@ManyToOne
 	private ClinicianPriority clinicianPriority;
 	
+	@NotNull
 	@ManyToOne
 	private PatientReadiness patientReadiness;
+	
```

Comment:
Would it make sense to combine referred / referralComplete into a single referralStatus field that took an enumerated set of values?

Reply:
The stakeholders were specific about wanting two columns, but we'll discuss in the next review with them. Agreed this is better as one column, as the table is becoming unsustainably wide. For me, there are also some concerns about a single date being attached to this entity without a real understanding about which fields have changed.